docs/guides/js-waku/relay-send-receive.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Send and Receive Messages Using Relay
+---
+
+This guide provides detailed steps to set up a Waku node for sending and receiving messages using the [Relay protocol](/overview/concepts/protocols#relay).
+
+## Create a Waku Node
+
+Set up a Waku node by creating a relay node, connecting to network peers, choosing a [content topic](/overview/concepts/content-topics), and creating an `encoder` and `decoder` for [message encryption](https://rfc.vac.dev/spec/26/):
+
+```js
+import {
+	createRelayNode,
+	waitForRemotePeer,
+	createEncoder,
+	createDecoder
+} from "@waku/sdk";
+
+// Create and start a relay node
+const node = await createRelayNode({
+	defaultBootstrap: true, // bootstraps using pre-defined nodes
+	emitSelf: true, // emits sent message events to itself
+});
+await node.start();
+
+// Wait for a successful peer connection
+await waitForRemotePeer(node);
+
+// Choose a content topic
+const contentTopic = "/relay-guide/1/message/proto";
+
+// Create a message encoder and decoder
+const encoder = createEncoder({ contentTopic });
+const decoder = createDecoder(contentTopic);
+```
+
+:::info
+The `emitSelf` option emits sent message events to itself and invokes the node's subscribers.
+:::
+
+## Create a Message Structure
+
+Create your application's message structure using [Protobuf's valid message](https://github.com/protobufjs/protobuf.js#usage) fields:
```

Comment:
usage of the word _valid_ sounds strange here. What did you mean?

Reply:
"valid message" is a protobuf.js concept

<img width="832" alt="Screenshot 2023-07-20 at 13 15 26" src="https://github.com/waku-org/docs.waku.org/assets/47832826/d5f9a28d-0a56-41b6-bb6f-eafaf856b481">

https://github.com/protobufjs/protobuf.js#valid-message